example/config/server/server.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+	"runtime/debug"
+
+	todo "github.com/vektah/gqlgen/example/config"
```

Comment:
aliased todo?

Reply:
ya, I copied from example/todo . I thought that `config` was a bit confusing